### DISCUSSION TITLE
General questions about Validity

### DISCUSSION THREAD

**Discussion by amyasnikov** [amyasnikov/validity#42] [Q&A]
Here you can ask a question related to Validity usage.
If you want to report a bug or propose new functionality consider using [Issues](https://github.com/amyasnikov/validity/issues)

**Comment by PieterL75** [comment]
Looking forward to getting to know Validity. Looks like the tool I've been waiting for 👍

**Reply by amyasnikov** [reply]
Nice to hear it from you, feel free to ask questions

**Reply by 601an** [reply]
Hello,

Is it possible to run the tests/comparisons outside the validity plugin/netbox and bulk upload the results so they can be visualized in the results ui?

**Reply by amyasnikov** [reply]
Hey @601an,
I don't understand your case well enough, but If you run your tests by some other tool (not validity), it would be definitely easier to implement some vizualization by yourself rather than trying to fit your data into validity/netbox data model.

Technically the plugin does not allow users to create/modify Test Results from the UI/API, but of course you still can write ORM or SQL queries which insert the required data into NetBox DB directly.
